frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -186,27 +186,29 @@ const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTableProps> = (
     </div>
     <div className="co-m-pane__body">
       <SectionHeading text="Update History" />
-      <div className="co-table-container">
-        <table className="table">
-          <thead>
-            <tr>
-              <th>Version</th>
-              <th>State</th>
-              <th>Started</th>
-              <th>Completed</th>
-            </tr>
-          </thead>
-          <tbody>
-            {_.isEmpty(history) && <EmptyBox label="History" />}
-            {history.map((update, i) => <tr key={i}>
```

Comment:
If we just use `_.map` here it will inherently handle the case where `updates` is empty or undefined.